### PR TITLE
Add the missing v122 v131 v132 to upgrade-matrix

### PR DESCRIPTION
**Pull request by w13915984028**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**

v122, v131, v132 images lists are not packed into v140 iso, the issue https://github.com/harvester/harvester/issues/5749 can not be validated.

**Solution:**

Add those missing versions to upgrade-matrix.

**Related Issue:**
https://github.com/harvester/harvester/issues/6620
https://github.com/harvester/harvester/issues/5749

**Test plan:**

via validate https://github.com/harvester/harvester/issues/6620

### CODE REVIEW

**Comment by w13915984028** [comment]
@mergify backport v1.4

**Comment by w13915984028** [comment]
@mergify backport v1.3

**Comment by bk201** [comment]
The file is OK on our tag commits, but reflecting the change in all branches is nice. Good spot.

**Review by starbops — APPROVED** [review]
LGTM, thank you.

File: package/upgrade-matrix.yaml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 versions:
+- name: v1.3.2
+  minUpgradableVersion: v1.2.2
```
Comment (bk201) [inline package/upgrade-matrix.yaml]:
```suggestion
  minUpgradableVersion: v1.3.1
```

File: package/upgrade-matrix.yaml [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+- name: v1.3.1
+  minUpgradableVersion: v1.2.2
 - name: v1.3.0
   minUpgradableVersion: v1.2.2
+- name: v1.2.2
+  minUpgradableVersion: v1.1.2
```
Comment (bk201) [inline package/upgrade-matrix.yaml]:
```suggestion
  minUpgradableVersion: v1.2.1
```

**Review by bk201 — CHANGES_REQUESTED** [review]

**Comment by w13915984028** [comment]
@bk201 For v1.3.1, is the minUpgradableVersion v1.2.2 or v1.3.0?

**Review by bk201 — APPROVED** [review]

**Comment by bk201** [comment]
minVersion of 1.3.1 is 1.2.2.

**Comment by w13915984028** [comment]
Code is rebased to have additional chance to validate PR https://github.com/harvester/harvester/pull/6629 as well.

The new build and CI PASS.

**Pull request merged**